{"version":"3.8.0","messages":["\nAn error occurred while loading ./spec/first_challenge_spec.rb.\n\u001b[31mFailure/Error: \u001b[0mremove(contacts)\u001b[0m\n\u001b[31m\u001b[0m\n\u001b[31mNameError:\u001b[0m\n\u001b[31m  undefined local variable or method `contacts' for main:Object\u001b[0m\n\u001b[31m  Did you mean?  context\u001b[0m\n\u001b[36m# ./lib/first_challenge.rb:27:in `<top (required)>'\u001b[0m\n\u001b[36m# ./spec/first_challenge_spec.rb:2:in `require_relative'\u001b[0m\n\u001b[36m# ./spec/first_challenge_spec.rb:2:in `<top (required)>'\u001b[0m\n","\nAn error occurred while loading ./spec/second_challenge_spec.rb.\n\u001b[31mFailure/Error: \u001b[0mflatten(groceries)\u001b[0m\n\u001b[31m\u001b[0m\n\u001b[31mNameError:\u001b[0m\n\u001b[31m  undefined local variable or method `groceries' for main:Object\u001b[0m\n\u001b[36m# ./lib/second_challenge.rb:12:in `<top (required)>'\u001b[0m\n\u001b[36m# ./spec/second_challenge_spec.rb:2:in `require_relative'\u001b[0m\n\u001b[36m# ./spec/second_challenge_spec.rb:2:in `<top (required)>'\u001b[0m\n","No examples found."],"examples":[],"summary":{"duration":0.000858408,"example_count":0,"failure_count":0,"pending_count":0,"errors_outside_of_examples_count":2},"summary_line":"0 examples, 0 failures, 2 errors occurred outside of examples"}